techniques/system/common/1.0/rudder-stdlib-core.cf

```diff
@@ -1,285 +0,0 @@
-#####################################################################################
-# Copyright 2012-2014 Normation SAS
-#####################################################################################
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, Version 3.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#####################################################################################
-#
-# Rudder Promise Body and Bundle Library - core part
-#
-# This library includes standardized bundles and bodies to be used as part of the
-# "best practices" in the Techniques writing, for classes and reporting only.
-# Other bodies and bundles should be included in rudder-stdlib.st
-#
-
-##################################################
-# classes bodies
-##################################################
-
-#
-# Automatically defines classes bases on a given prefix
-# The classes are defined based on the romises outcome
-#
-body classes rudder_common_classes(prefix)
```

Comment:
this is used in many places

Reply:
aah migrated to ncf

Reply:
Yes, this changes is parallel to https://github.com/Normation/ncf/pull/887